framework/packages/abstract-std/src/objects/gov_type.rs

```diff
@@ -27,13 +27,16 @@ pub enum GovernanceDetails<T: AddressLike> {
         // Account address
         account: T,
     },
-    /// An external governance source
+    /// An external governance source. This could be a cw3 contract for instance
+    /// The External Governance source still leverages one address that is admin of the contract
```

Comment:
```suggestion
    /// An external governance source. This could be a cw3 contract for instance
    /// The `governance_address` will be the admin of the Account.
```